Xamarin.Forms.Platform.WPF/Renderers/EntryRenderer.cs

```diff
@@ -24,6 +24,7 @@ protected override void OnElementChanged(ElementChangedEventArgs<Entry> e)
 					Control.LostFocus += OnTextBoxUnfocused;
 					Control.TextChanged += TextBoxOnTextChanged;
 					Control.KeyUp += TextBoxOnKeyUp;
+					Control.SelectionChanged += TextBoxOnSelectionChanged;
```

Comment:
Unsubscribe this event subscription doing the renderer dispose https://github.com/xamarin/Xamarin.Forms/blob/84372adc48b7eac0efaa91f84544692325faa660/Xamarin.Forms.Platform.WPF/Renderers/EntryRenderer.cs#L287

Reply:
Done